static/docs/use-cases/index.md

```diff
@@ -1,15 +1,28 @@
 # Use Cases
 
-Here we provide an overview for some DVC use cases, from very basic
-([data and model files management](/doc/use-cases/data-and-model-files-versioning))
-to more advanced (optimizing resources on a
-[single development machine](/doc/use-cases/multiple-data-scientists-on-a-single-machine)).
-This list is _not_ exhaustive. We keep growing it and try to include interesting
-scenarios from the community. Please, [contact us](/support) if you have some
-suggestions!
-
-If you want to get some hands-on experience we recommend to follow along
-[Get Started](/doc/get-started) and try examples at the very end of it. They
-include prepared datasets and code to run. Use cases are not written to be run
-end to end, specifically we don't provide prepared datasets and code to train
-models.
+We provide short articles on common DVC scenarios in this section, with a
+justified context (usually extracted from real-life support cases). Their
+solution approaches combine several features and commands of DVC, for didactic
+purposes. These range from basic to more advanced:
+
+- [Data Versioning](/doc/use-cases/data-and-model-files-versioning) describes
+  our most primary use: tracking and versioning large files with Git + DVC.
+- [Share Data and Model Files](/doc/use-cases/share-data-and-model-files) goes
+  over basic collaboration possibilities enabled by DVC.
+- [Update a Tracked File](/doc/use-cases/update-tracked-files) explains how to
+  modify protected data under DVC control (related to the `cache.type` option of
+  DVC).
+- [Shared Development Server](/doc/use-cases/shared-development-server.md)
+  describes a single development machine setup for teams that prefer so.
+
+This list of use cases is _not_ exhaustive. We keep reviewing our docs and will
+include interesting scenarios that surface in our community. Please,
+[contact us](/support) if you need help or have suggestions!
+
+Use cases are not written to be run end-to-end. For more general hands-on
+experience with DVC, we recommend to follow the [Get Started](/doc/get-started),
+and/or [Tutorial](/doc/tutorial)] first – These include prepared datasets,
```

Comment:
These -> these?

Reply:
Changing to "which" (lower case, yes), actually. Thanks

Reply:
Updates to this file in 2e26095.